test/openS3_v3.js

```diff
@@ -0,0 +1,35 @@
+const test = require("tap").test;
+const unzip = require("../unzip");
+
+const version = +process.version.replace("v", "").split(".")[0];
+
+test(
+  "get content of a single file entry out of a zip",
+  { skip: version < 16 },
+  function (t) {
+    const { S3Client } = require("@aws-sdk/client-s3");
+
+    const client = new S3Client({
+      region: "us-east-1",
+      signer: { sign: async (request) => request },
+    });
+
+    // These files are provided by AWS's open data registry project.
+    // https://github.com/awslabs/open-data-registry
+
+    return unzip.Open.s3_v3(client, {
```

Comment:
If there's a better file source for this I can change it. The one referenced by `openS3.js` no longer exists.